kumascript/macros/geckoRelease.ejs

```diff
@@ -2,10 +2,10 @@
 
 /* Appending a "+" to the version number you specify means that version and all later versions. */
 var geckoRelease = '';
-var arg = string.trim($0);
+var arg = ($0 || "").trim();
```

Comment:
Feel free to use
```suggestion
var arg = `$0`.trim();
```
which is the more common "modern" way of doing it. Incidentally, in Babel, it gets converted to ES5 as basially what you had.

Reply:
I'm not sure if <code>\`$0\`</code> will actually resolve `$0` or treat it as string `'$0'`. To avoid ambiguity, I used <code>\`${$0}\`</code>.